File: functions/swap.c
#include <stdio.h>
void swap(){
	int a,b,c;
	printf("Give value of a: \n");
	scanf("%d", &a);
	printf("Give value of b: \n");
	scanf("%d", &b);
	printf("Value of a before swap is: %d \n",a);
	printf("Value of b before swap is: %d \n", b);
	c=a;
	a=b;
	b=c;
	printf("Value of a after swap is: %d \n",a);
	printf("Value of b after swap is: %d \n", b);
}
int main(void) {
	swap();
	
	
	return 0;
}

File: JAVA_ASSIGNMENT/swap.java
import java.util.*;
class pane{
    public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.println("Enter value1: ");
    int value1 = sc.nextInt();
    System.out.println("Enter value2: ");
    int value2 = sc.nextInt();
    System.out.println("BEFORE SWAP");
    System.out.println("Value1: "+value1+"\nvalue2: "+value2);
    System.out.println("AFTER SWAP");
    int temp = value1;
    value1 = value2;
    value2 = temp;
    System.out.println("Value1: "+value1+"\nvalue2: "+value2);
    }
}
